client/src/styles/bundle.scss

```diff
@@ -63,6 +63,7 @@
 @import "../components/Modal/Modal";
 @import "../components/TreeDropdownField/TreeDropdownField";
 @import "../components/TinymceInlineToolbar/TinymceInlineToolbar";
+@import "../components/BrowserWarning/BrowserWarning";
```

Comment:
should this be imported in the main bundle if it's already getting its own separate output file?
I'd suggest including the css file in the template which includes the separate javascript file as well.